README.md

```diff
@@ -2,31 +2,37 @@
 
 Command line client and gem for modifying and reading from the stronghold configuration store
 
-## Installation
+## Using the command-line client, `stronghold-cli`
 
-Add this line to your application's Gemfile:
+To install:
 
-````ruby
-gem 'stronghold'
+````bash
+git clone git@github.com:pusher/stronghold-ruby.git
+cd stronghold-ruby
+gem build stronghold
+gem install stronghold
 ````
 
-And then execute:
+To use:
 
-````bash
-bundle
+````
+stronghold-cli (--path /path/ |--app app | --list) [operation]
 ````
 
-Or install it yourself as:
+## Using the `stronghold` Ruby library
 
-````bash
-gem install stronghold
+Add this line to your application's Gemfile:
+
+````ruby
+gem 'stronghold'
```

Comment:
I think you would have to add the git repository since this is not on https://rubygems.org/

Reply:
👍 I've updated, copied from this random thing https://github.com/pusher/stronghold-yaml-dump/blob/1f0886b61ff4e09c83ab3cec48aedd4d8659159f/Gemfile#L3